.github/workflows/ci.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        node: ['18', '20', '21']
+        node: ['18', '20', '22']
```

```suggestion
        node: ['18', '20', '21', '22']
```

I believe that dropping node 21 from CI is a breaking change (unless we consider node 21 itself unstable), because otherwise we can break node 21 support in the next patch release and won't notice it. We talked about odd numbered nodejs versions in another PR, but I can't quite recall the outcome 🤔

---

I think the probability of breaking Node 21 only, while still running well on Node 20 and Node 22 is extremely low.

Having said that, I am fine running our CI on Node 21 as long as it's in maintenance mode.